File: src/block/AirBlock.java
package block;

public class AirBlock extends Block {
    protected AirBlock() {
        super("air");
    }

    @Override
    public boolean isFullCube(BlockState block) {
        return false;
    }

    @Override
    public BlockModel getModel(BlockState blockState) {
        return null;
    }
}

File: java/src/block/AirBlock.kt
package src.block

import src.block.Block
import src.block.BlockModel
import src.block.BlockState

class AirBlock : Block("air") {
    override fun getModel(blockState: BlockState): BlockModel {
        return BlockModel(arrayOf())
    }

    override fun hasModel(blockState: BlockState): Boolean {
        return false
    }
}